admin_app/src/app/content/edit/page.tsx

```diff
@@ -80,35 +94,43 @@ const AddEditContentPage = () => {
 const ContentBox = ({
   content,
   setContent,
+  content_id
+
 }: {
   content: Content | null;
   setContent: React.Dispatch<React.SetStateAction<Content | null>>;
+  content_id: number | null;
 }) => {
-  const [isSaved, setIsSaved] = React.useState(true);
+  const [isSaved, setIsSaved] = React.useState(false);
```

Comment:
I updated it to false to send a message whenever a content is succesfully updated to DB